include/swift/AST/LocalizationLanguages.def

```diff
@@ -0,0 +1,27 @@
+//===--- LocalizationLanguages.def - Localization Languages ----*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file has all languages currently supported in localization.
+//
+//===----------------------------------------------------------------------===//
+
+#define DEFINE_LOCALIZATION_MACROS
+#include "DefineLocalizationMacros.h"
+
+#define LOCALE_NO_UNDEF
+
+LOCALE(en, English)
```

Comment:
I only added English for now. Should I add all of the languages?

Reply:
Yeah, let's add more before otherwise it would be impossible to test localizations.